.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+<!--
+A high-level description of what the bug is. Make sure to clearly and concisely describe where the bug was encountered, what happened and what you expected to happen. Provide more details on each of these in the sections below.
+-->
+
+**To Reproduce**
+<!--
+Provide the information necessary to reproduce the bug. If the bug occurred when executing ``skbase`` related code, provide a reproducible code example. If the bug arose in another context, provide instructions for recreating the bug.
```

Comment:
maybe worth saying: optimally, the code should not rely on external data sources, and should be minimally reproducible, perhaps also a link to sth like this
https://stackoverflow.com/help/minimal-reproducible-example

Reply:
That is what we want. Makes sense to be more direct indicating that.